io-engine/src/grpc/v1/replica.rs

```diff
@@ -106,6 +110,21 @@ impl Default for ReplicaService {
     }
 }
 
+impl From<VolumeSnapshotDescriptor> for ReplicaSnapshot {
+    fn from(s: VolumeSnapshotDescriptor) -> Self {
+        Self {
+            snapshot_uuid: s.snapshot_uuid().to_string(),
+            snapshot_name: s.snapshot_params().name().unwrap_or_default(),
+            replica_uuid: s.replica_uuid().to_string(),
+            bytes_referenced: s.bytes_referenced(),
+            num_clones: s.num_clones(),
+            timestamp: Some(s.timestamp().into()),
```

Comment:
Can timestamp here be None (it's an Option) ? If no, maybe we can make it a regular non-Option field ?

Reply:
> Can timestamp here be None (it's an Option) ? If no, maybe we can make it a regular non-Option field ?

We can keep it Option field to sync with other gRPC implementation. Now Will set to `None`.  Based on this field update, We will change the value in next PR.